charts/enterprise-logs/templates/admin-api/deployment-admin-api.yaml

```diff
@@ -41,8 +41,10 @@ spec:
       {{- if .Values.adminApi.priorityClassName }}
       priorityClassName: {{ .Values.adminApi.priorityClassName }}
       {{- end }}
+      {{- if .Values.adminApi.securityContext.runAsGroup }}
       securityContext:
-        fsGroup: 10001
+        fsGroup: {{ .Values.adminApi.securityContext.runAsGroup | int }}
+      {{- end}}
```

Comment:
In case, `runAsGroup` is not defined, the `fsGroup`, is not written. Shouldn't we default for that case?
```
securityContext:
{{- if .Values.adminApi.securityContext.runAsGroup }}
  fsGroup: {{ .Values.adminApi.securityContext.runAsGroup | int }}
{{- else }}
  runAsGroup: 10001
  fsGroup: 10001
{{- end}}
```

Reply:
If `runAsgroup` is not defined in the values.yaml then also the securityContext on the container will not be set, see https://github.com/grafana/helm-charts/blob/main/charts/enterprise-logs/templates/admin-api/deployment-admin-api.yaml#L118-L120
However, it probably won't work unless you use a modified Loki Docker image.